packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -77,25 +104,181 @@ function WorklistDataGrid(props) {
     rows,
     columnHeaderHeight,
     onColumnSelect,
+    onCellSelect,
+    onRowSelect,
+    onRowSelectAll,
+    onClearSelectedRows,
+    onDisableSelectableRows,
+    hasSelectableRows,
     intl,
+    rowHeaderIndex,
   } = props;
 
+  const grid = useRef();
   const focusedRow = useRef(0);
   const focusedCol = useRef(0);
-  const grid = useRef();
+  const ariaLiveMsg = useRef();
+
+  const [currentSelectedCell, setCurrentSelectedCell] = useState(null);
+  const [, setRowSelectionModeMessage] = useState();
```

Comment:
Why is there no corresponding state variable, eg `[rowSelectionModeMessage, setRowSelectionModeMessage]`.

Reply:
The variable is not used and the linter will produced an error if defined.